build.xml

```diff
@@ -5,26 +5,26 @@
     <property name="dist_dir" value="${basedir}"/>
     <property name="tests_dir" value="${basedir}/tests"/>
     <property name="reports_dir" value="${basedir}/reports"/>
-    <property name="version" value="2.1.8"/>
+    <property name="version" value="3.0.0"/>
```

Comment:
Do we still want to keep this build.xml around?  Can it be removed now that we're using maven?

Reply:
Yes for 2.1.9, no for anything that comes afterwards.

Reply:
And speaking of it, after 2.1.9, I'll clean the file/directory layout to standard `src/{main,test}/java`.